apps/dashboard/src/lib/utils/store/org.ts

```diff
@@ -16,6 +16,7 @@ export const defaultCurrentOrgState: CurrentOrg = {
   memberId: '',
   role_id: '',
   landingpage: {},
+  customization: {},
```

Comment:
@Chifez Why did you create a different store for the customizations when there is already a store that holds the `customization` in the `currentOrg`?

The store is: `apps/dashboard/src/lib/components/Org/Settings/store.ts`